packages/bot-web-ui/src/components/dashboard/bot-builder/toolbar/toolbar.tsx

```diff
@@ -0,0 +1,252 @@
+import { Button, Icon, Popover, Dialog } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import React from 'react';
+import LoadModal from 'Components/load-modal';
+import SaveModal from 'Components/save-modal';
+import { tabs_title } from 'Constants/bot-contents';
+import { popover_zindex } from 'Constants/z-indexes';
+import { connect } from 'Stores/connect';
+import RootStore from 'Stores/index';
+
+interface IconButtonProps {
+    popover_message: string;
+    icon: string;
+    icon_id: string;
+    iconOnClick: () => void;
+    icon_color?: string;
+}
+
+const IconButton = ({ popover_message, icon, icon_id, icon_color, iconOnClick }: IconButtonProps) => (
```

Comment:
I think there are many components in 1 file ... can we separate the in separate files? and if they are the components that might be used later for other components maybe we can put them under the dashboard-component folder and only keep the toolbar implementation here.

Reply:
I will do it in the next PR